src/pages/ProductDetail.js

```diff
@@ -1,46 +1,125 @@
 import { useEffect, useState } from 'react';
 import Product from '../components/Product/Product';
 import RegularInfo from '../components/RegularInfo';
+import Count from '../components/Count/Count';
+import ProductInfo from '../components/ProductInfo';
+import { useParams } from 'react-router-dom';
 import './ProductDetail.scss';
 
 const ProductDetail = () => {
-  const name = '글림체 자음모음 붙임딱지';
-  const price = 3500;
-  const number = 1;
+  const params = useParams();
+  const productID = params.id;
 
   const [carouselDatas, setCarouselData] = useState([]);
+  // const [goToCart, setGoToCart] = useState([]);
+  const [product, setProduct] = useState([]);
   const [currentTab, setCurrentTab] = useState('First');
+  const [imgChange, setImgChange] = useState(false);
+  const [number, setNumber] = useState(1);
 
   useEffect(() => {
-    fetch('./data/MainData.json')
+    fetch('/data/MainData.json')
       .then(response => response.json())
       .then(data => setCarouselData(data));
   }, []);
 
+  useEffect(() => {
+    fetch('/data/DetailData.json')
+      .then(response => response.json())
+      .then(result => {
+        setProduct(result[0]);
+      });
+  }, []);
+
+  // console.log('product', product);
+  // console.log(carouselDatas);
+
+  const goToCart = () => {
+    fetch('./carts', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: localStorage.getItem('accessToken'),
```

Comment:
```suggestion
const token = localStorage.getItem('accessToken')
// 코드생략
        Authorization: token,
```
위와 같이 변수로 선언해 놓으시면 POST요청에서 뿐 만 아니라 다른곳에서 토큰을 사용할 때 편리하게 사용할 수 있습니다